python/python/glide/async_commands/server_modules/json.py

```diff
@@ -193,6 +223,66 @@ async def arrlen(
     )
 
 
+async def arrpop(
+    client: TGlideClient,
+    key: TEncodable,
+    options: Optional[JsonArrPopOptions] = None,
+) -> Optional[TJsonResponse[bytes]]:
+    """
+    Pops the last element from the array located at the specified path within the JSON document stored at `key`.
+    If `options.index` is provided, it pops the element at that index instead of the last element.
+
+    See https://valkey.io/commands/json.arrpop/ for more details.
+
+    Args:
+        client (TGlideClient): The client to execute the command.
+        key (TEncodable): The key of the JSON document.
+        options (Optional[JsonArrPopOptions]): Options including the path and optional index. See `JsonArrPopOptions`.
+
+    Returns:
+        Optional[TJsonResponse[bytes]]:
+            For JSONPath (`path` starts with `$`):
+                Returns a list of bytes string replies for every possible path, representing the popped JSON values,
+                or None for JSON values matching the path that are not an array or are an empty array.
+                If a value is not an array, its corresponding return value is null.
+            For legacy path (`path` starts with `.`):
+                Returns a bytes string representing the popped JSON value, or None if the array at `path` is empty.
```

Comment:
Mention: in that case all matched arrays are modified even though only 1 value returned